client/src/ui-components/ClientOrdersComponent/ClientOrderTableRow.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import "./ClientOrderTableRow.css";
+function ClientOrderTableRow(props) {
+  const [status, setStatus] = useState(props.order.status);
+
+  const handleSubmit = () => {
+    if (status === "WAITING") {
+      setStatus("COMPLETED");
+    } else if (status === "COMPLETED") {
+      setStatus("WAITING");
+    }
+  };
```

Comment:
Watch on the API document the right status:
Initial status: PREPARED
Final: DONE